src/players/shmup/plugin.rs

```diff
@@ -11,7 +12,7 @@ pub struct ShmupPlugin;
 impl Plugin for ShmupPlugin {
     fn build(&self, app: &mut App) {
         // set GitHub dark mode background color
-        let bg_color: Color = Color::hex("22272d").unwrap();
+        let bg_color: Color = Color::hex("2d333b").unwrap_or_default();
```

Comment:
I changed the background color because one of the entities had a very dark color which made it barely visible